CONTRIBUTING.md

```diff
@@ -1,13 +1,14 @@
 ## Welcome!
 
-We’re so glad you’re thinking about contributing to a Technology Transformation Services (TTS) open source project! If you’re unsure about anything, just ask — or submit your issue or pull request anyway. The worst that can happen is we’ll politely ask you to change something. We appreciate all friendly contributions.
+Thanks for thinking about contributing to a Technology Transformation Services (TTS) open source project! If you’re unsure about anything, just ask.
 
 TTS is committed to building a safe, welcoming, harassment-free culture for everyone. We expect everyone on the TTS team and everyone within TTS spaces, including contributors to our projects, to follow the [TTS Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md).
 
-We encourage you to read this project’s CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md). When you are ready to make a pull request, read our [pull request process](https://handbook.login.gov/articles/pull-request-review.html), which is a part of [the Login.gov Handbook](https://handbook.login.gov/).
+If you are an external contributor, please check out our [External Contributor Guidelines](EXTERNAL_CONTRIBUTORS.md) before beginning any work on a code change.
```

Comment:
This might be nit pick, but would it be helpful to bold the part about an external contributor so that eyes can be draw to that paragraph?
```suggestion
**If you are an external contributor**, please check out our [External Contributor Guidelines](EXTERNAL_CONTRIBUTORS.md) before beginning any work on a code change.
```